requirements-dev.txt

```diff
@@ -7,5 +7,3 @@ requests-html==0.10.0
 PyYAML==5.3.1
```

Comment:
Already on "install_requires" as setup.cfg. So these 3 will be installed automatically.
Are this only needed for dev? In that case we can remove them from "install_requires".

Reply:
I think it should be kept in for the developers because they're two seperate systems that don't really use each other. `setup.cfg`/`setup.py` is only used if you run `pip install .` which you run if you want to install the application to actually use it, but as far as I know, when developing you just install requirements with `pip install -r requirements.txt` and your IDE uses that. You probably don't want to actually `install` the applications elsewhere when developing since you're working on it in the project folder.

Reply:
On that note, to avoid repeating the names/versions of the libraries, I've found a way that we can have requirements.txt install the requirements from setup.cfg/.py: using `-e .` (https://stackoverflow.com/a/49684835). This _does_ install the application (and ofc. the dependencies), but symlinks it to the development directory so you can properly continue developing without having to repeatedly install it. I'll make a commit with this.